package.json

```diff
@@ -5,13 +5,14 @@
     "cheerio": "^1.0.0-rc.10",
     "easy-zip": "0.0.4",
     "ejs": "2.5.5",
+    "esm": "^3.2.25",
```

Comment:
you seems to have upgrade the project toe module. do you still need this module?

Reply:
no, fixed